templates/apps/inbox.html

```diff
@@ -1,48 +1,69 @@
 {% extends "apps/_base.html" %}
 
 {% block inner %}
-  <div class="form-group">
+<div class="form-group">
     {% for app in apps %}
 
     <div class="card">
-      <div class="card-block">
-        <h4 class="card-title"><a href="{% url 'show_application' app.id %}">{{app.first_name}} {{app.last_name}}</a></h4>
-        <h6 class="card-subtitle mb-2 text-muted">{{app.gender}} für {{app.country}}</h6>
-        <p class="card-text"><strong>Motivation</strong></p>
-        <p class="card-text">{{app.motivation|linebreaks }}</p>
-        <p class="card-text"><strong>Fähigkeiten</strong></p>
-        <p class="card-text">{{app.skills|linebreaks }}</p>
-        <p class="card-text"><strong>Ethik-Frage</strong></p>
-        <p class="card-text">{{app.ethical_dilemma|linebreaks }}</p>
-        {% if app.internet_profiles %}
-          <p class="card-text"><strong>Internet Profile</strong></p>
-          <p class="card-text">{{app.internet_profiles|linebreaks }}</p>
-        {% endif %}
-        {% if app.affiliations %}
-          <p class="card-text"><strong>Partei- oder Interessenangehörigkeit</strong></p>
-          <p class="card-text">{{app.affiliations|linebreaks }}</p>
-        {% endif %}
-        {% if app.marktplatz_name %}
-          <p class="card-text"><strong>Marktplatz Konto</strong> <a href="https://marktplatz.bewegung.jetzt/u/{{app.marktplatz_name}}/summary">{{app.marktplatz_name}}</a></p>
-          <p class="card-text marktplatz-info"></p>
-        {% endif %}
-      </div>
-
-      <div class="card-block">
-        <form action="/vote/{{app.id}}" method="POST">
-          {% csrf_token %}
-          <textarea name="comment" placeholder="Kommentar (optional)"></textarea>
-          <button class="btn btn-success" type="submit" name="vote" value="y">🙂</button>
-          <button class="btn btn-danger nope-button" type="submit" name="vote" value="n">😕</button>
-        </form>
-      </div>
+        <div class="card-block">
+            <div class="card-counter">
+                {{ forloop.counter }} / {{ apps|length }}
+            </div>
+            <h4 class="card-title">
+                <a href="{% url 'show_application' app.id %}">
+                    {{app.first_name}}
+                    {{app.last_name}}
+                </a>
+            </h4>
+            <h6 class="card-subtitle mb-2 text-muted">{{app.gender}} für {{app.country}}</h6>
+            <p class="card-text"><strong>Motivation</strong></p>
+            <p class="card-text">{{app.motivation|linebreaks }}</p>
+            <p class="card-text"><strong>Fähigkeiten</strong></p>
+            <p class="card-text">{{app.skills|linebreaks }}</p>
+            <p class="card-text"><strong>Ethik-Frage</strong></p>
+            <p class="card-text">{{app.ethical_dilemma|linebreaks }}</p>
+            {% if app.internet_profiles %}
+            <p class="card-text"><strong>Internet Profile</strong></p>
```

Comment:
any particular reason you removed the indentation within `{%if %}`-clauses? I think it is helps to figure out which blocks belong too it, don't you?

Reply:
no particular reason. just <ALT>+<UMSCH>+<F> within my ide to format code automatically. O.o
i fixt this. please have also a look at my additional commits. i added a missing validation to my code.